### ISSUE TITLE
NPE OnDatePicked in CenterListFragment (Centers Collection Sheet Offline feature)

### ISSUE THREAD

**Issue by therajanmaurya** [openMF/android-client#296]
Centers Collection Sheet Offline feature

``` java

FATAL EXCEPTION: main
Process: com.mifos.mifosxdroid, PID: 16231
java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.Integer.intValue()' on a null object reference
at com.mifos.mifosxdroid.online.centerlist.CenterListFragment$3.onDatePicked(CenterListFragment.java:208)
at com.mifos.mifosxdroid.uihelpers.MFDatePicker.onDateSet(MFDatePicker.java:74)
at android.app.DatePickerDialog.onClick(DatePickerDialog.java:137)
at com.android.internal.app.AlertController$ButtonHandler.handleMessage(AlertController.java:163)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5443)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)

```

**Comment by therajanmaurya** [comment]
@ishan1604 Please look this scenario and suggest me.

Center Collection Sheet is dependent to the [demo.openmf.org/fineract-provider/api/v1/centers/{centerId}?associations=groupMembers,collectionMeetingCalendar](demo.openmf.org/fineract-provider/api/v1/centers/{centerId}?associations=groupMembers,collectionMeetingCalendar).

response collectionMeetingCalendar, Which is not coming from REST API for all centers. for some centers, collectionMeetingCalendar object is not coming from REST API. So the app is crashing during genrating collection sheet.

**Comment by sosnickm** [comment]
Is this still a problem?

**Comment by therajanmaurya** [comment]
Yes, If you wanna work, you can. 
Feel free to ask any question regarding the issue!

**Comment by top-gun007** [comment]
@therajanmaurya can you tell me more about this issue..thank you

**Comment by therajanmaurya** [comment]
Fixed in #440
